### DISCUSSION TITLE
SEB and automatic updates?

### DISCUSSION THREAD

**Discussion by kvet-divizny** [SafeExamBrowser/seb-win-refactoring#877] [General]
Hello, I wanted to ask if SEB has an automatic update feature. We have encountered an issue due to an unexpected update of SEB and are trying to determine how this update occurred. Does SEB use automatic updates? If yes, can this feature be disabled?

**Comment by dbuechel** [comment]
No, SEB does not have an automatic update feature and most likely will never have one (except the iOS version, as far as I am aware).
